src/pages/lessons/[slug].tsx

```diff
@@ -276,13 +273,18 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
         if (!media) {
           send('LOAD')
         }
-
         break
       case 'completed':
-        completeVideo()
+        console.debug('handling a change to completed', {lesson})
+        onEnded(lesson).then((lessonView: any) => {
```

Comment:
when this callback was passed into the player event handler below it was always using a different instance of `lesson` which leads me to believe there might be multiple player instances?

this was very unpleasant to debug.